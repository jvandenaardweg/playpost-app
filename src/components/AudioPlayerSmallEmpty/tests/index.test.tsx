import React from 'react';
import { render, RenderAPI } from 'react-native-testing-library';

import { AudioPlayerSmallEmpty } from '../index';

describe('AudioPlayerSmallEmpty', () => {

  describe('rendering', () => {
    let wrapper: RenderAPI;

    beforeAll(() => {
      wrapper = render(<AudioPlayerSmallEmpty />);
    });

    it('should render correctly', () => {
      expect(wrapper.toJSON()).toMatchSnapshot();
    });
  });
});
