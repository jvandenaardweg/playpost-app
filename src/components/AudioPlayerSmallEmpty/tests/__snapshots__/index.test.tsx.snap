// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayerSmallEmpty rendering should render correctly 1`] = `
<View
  style={
    Object {
      "height": 55,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#000000",
        "flexDirection": "row",
        "height": 55,
        "justifyContent": "space-around",
        "paddingBottom": 8,
        "paddingLeft": 16,
        "paddingRight": 16,
        "paddingTop": 8,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontSize": 15,
            "fontWeight": "600",
          }
        }
      >
        Select an article to listen
      </Text>
    </View>
  </View>
</View>
`;
